site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -247,9 +262,9 @@ chrome.action.onClicked.addListener(function(tab) {
 
 ### Omnibox {: #omnibox }
 
-Users can invoke extension functionality through the [omnibox][26]. Include the `"omnibox"` field in
-the manifest and designate a keyword. The [Omnibox New Tab Search][27] sample extension uses "nt" as
-the keyword.
+Users can invoke extension functionality through the [omnibox][api-omnibox]. Include the `"omnibox"`
+field in the manifest and designate a keyword. The [Omnibox New Tab Search][sample-new-tab-search]
+sample extension uses "nt" as the keyword.
```

Comment:
Slight tweak that (hopefully) makes it more obvious that the quotes are not required.

```suggestion
sample extension uses `nt` as the keyword.
```